From 345967dc8a276f36b5293e521595ccff56ac4377 Mon Sep 17 00:00:00 2001
From: lrinQVQ <cqlrin@gmail.com>
Date: Wed, 9 Jan 2019 16:33:21 +0800
Subject: [PATCH 2/2] ignore generated directory

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index 79deb06c7..cc9fa0d56 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,6 @@
 build/
+build2/
+.openssl/
 ssl/test/runner/runner
 *.swp
 *.swo
-- 
2.11.0

