From 0d6aa9354c9dac29caad055f6aa035c586afaafb Mon Sep 17 00:00:00 2001
From: lrinQVQ <lrin@qvq.network>
Date: Mon, 23 Dec 2019 08:36:43 -0500
Subject: [PATCH 2/2] Add position-independent code support for static library

Signed-off-by: lrinQVQ <lrin@qvq.network>
---
 .gitignore     | 2 ++
 CMakeLists.txt | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index a8581a591..0eb512b5e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,8 @@
 build/
 build2/
+build3/
 .openssl/
+lib/
 build32/
 build64/
 ssl/test/runner/runner
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 75bf9981b..c059f7a62 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -296,7 +296,7 @@ endif()
 
 add_definitions(-DBORINGSSL_IMPLEMENTATION)
 
-if(BUILD_SHARED_LIBS)
+if(BUILD_SHARED_LIBS OR BUILD_PIC)
   add_definitions(-DBORINGSSL_SHARED_LIBRARY)
   # Enable position-independent code globally. This is needed because
   # some library targets are OBJECT libraries.
-- 
2.24.1

